src/Spryker/Zed/Merchant/Persistence/Propel/Mapper/MerchantMapper.php

```diff
@@ -44,4 +50,54 @@ public function mapEntityToMerchantTransfer(
             true
         );
     }
+
+
+    /**
+     * @param \Orm\Zed\Merchant\Persistence\SpyMerchant[] $merchantEntities
+     * @param \Generated\Shared\Transfer\MerchantCollectionTransfer $merchantCollectionTransfer
+     *
+     * @return \Generated\Shared\Transfer\MerchantCollectionTransfer
+     */
+    public function mapMerchantCollectionToMerchantCollectionTransfer(
+        $merchantEntities,
```

Comment:
please add type for `$merchantEntities`
```suggestion
        array $merchantEntities,
```